moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -79,12 +79,7 @@ bool RuckigSmoothing::applySmoothing(robot_trajectory::RobotTrajectory& trajecto
     return false;
   }
 
-  auto ruckig_result = runRuckigInBatches(trajectory, ruckig_input);
```

Comment:
The "batches" feature is not working well. Let's put this on ice. See https://github.com/ros-planning/moveit2/issues/1960